client/src/components/Demo-1/Result.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactMarkDown from 'react-markdown';
+import styled from 'styled-components';
+import content from '@services/content';
+
+import { Paragraph, Link, themeSpacing } from '@datapunt/asc-ui';
+
+export interface IProps { }
+
+const StyledH2 = styled.h2`
+  margin-top: ${themeSpacing(2)};
+  margin-bottom: ${themeSpacing(3)};
+`;
+
+const StyledParagraph = styled(Paragraph)`
+  margin-top: ${themeSpacing(2)};
+  margin-bottom: ${themeSpacing(2)};
+`;
+
+const StyledUL = styled.ul`
+  margin-top: 0;
+`;
+
+
+const Result: React.FC<IProps> = () => {
```

Comment:
Ik heb in demo2 het hele result component vervangen voor een markdown component. De gehele content van dit component kan in Markdown. We moeten wel even kijken wat we doen met StyledH2 en StyledParagraph aangezien dit vaker voorkomt. Ik denk dat we hiervan geexporteerde styled-components moeten maken.